org.contour.Terminal.yml

```
@@ -0,0 +1,92 @@
+app-id: org.contour.Terminal
+name: Contour Terminal Emulator
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

this runtime version is END OF LIFE
```suggestion
runtime-version: '5.15-21.08'
```